samples_project/Assets/SampleViewer/Scripts/SampleSwitcher.cs

```diff
@@ -118,6 +118,16 @@ private void OnEnable()
     private void PopulateSampleSceneList()
     {
         SceneDropdown.options.Clear();
+
+#if !(USE_OPENXR_PACKAGE)
+        if (SceneList.Contains("VR-Sample")){ 
+             SceneList.Remove("VR-Sample");
+        }
+#else
+        if (!SceneList.Contains("VR-Sample")){ 
+             SceneList.Add("VR-Sample");
```

Comment:
Won't VR-Sample already be in the list? So this would add it twice?

Reply:
It wouldn't add it if it was already in the list, only if it "not" contains VR-Sample, I did this because if you were to remove the OpenXR package it would remove the scene from the list, but if you reinstalled OpenXR after that I want it to re-enable the scene.